File: cSharp/day27.cs

using System;
public static float deduct(
    float x,
    float y
) {
    return Math.Max(x - y, 0);
}


Console.WriteLine("Input first number(to be deducted):");
string input3 = Console.ReadLine();

Console.WriteLine("Input second number(deductor):");
string input4 = Console.ReadLine();

float.TryParse(input3, out float numInput3);
float.TryParse(input4, out float numInput4);

Console.WriteLine(
    "Result: " +
    deduct(numInput3, numInput4).ToString()
);

File: rust/day27.rs
fn deduct(x: f32, y: f32) -> f32 {
    let diff: f32 = x - y;
    return diff.max(0.0);
}


fn main() {   
    use std::io;
    
    let mut number1 = String::new();
    println!("Input first number: ");
    io::stdin()
        .read_line(&mut number1)
        .expect("Failed to read number");
    let n1 = number1.trim().parse().expect("Not a number");
    

    
    
    let mut number2 = String::new();
    println!("Input second number: ");
    io::stdin()
        .read_line(&mut number2)
        .expect("Failed to read number");
    let n2 = number2.trim().parse().expect("Not a number");


    let result:f32 = deduct(n1,n2);



    println!(
        "{}",
        "Result: ".to_owned() + 
        &result.to_string()
    );

}
